src/service/translate.js

```diff
@@ -709,6 +710,29 @@ function $translate($STORAGE_KEY, $windowProvider, $translateSanitizationProvide
 
   /**
    * @ngdoc function
+   * @name pascalprecht.translate.$translateProvider#useForceAsyncReload
+   * @methodOf pascalprecht.translate.$translateProvider
+   *
+   * @description
+   * If force async reload is enabled, async loader will be always called
+   * even if $translationTable already contains the language key.
+   *
```

Comment:
Hm, then I remember it wrong - didn't have time to look it up in the source. My fault. Whoops :-) But then please add the documentation just the way around so it is clear what will happen for all people taking a look in here.